test/powershell/Modules/Microsoft.PowerShell.Management/UnixStat.Tests.ps1

```diff
@@ -0,0 +1,127 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+Describe "UnixFileSystem additions" -Tag "CI" {
+    # if PSUnixFileStat is converted from an experimental feature, these tests will need to be changed
+    BeforeAll {
+        $experimentalFeatureName = "PSUnixFileStat"
+        $skipTest = $true
+        if ( Get-ExperimentalFeature -Name $experimentalFeatureName ) {
+            $skipTest = $false
+        }
```

Comment:
This can be:

$skipTest = -not $EnabledExperimentalFeatures.Contains('PSUnixFileStat')

Reply:
fixed!